cmd/server/cmd/server.go

```diff
@@ -125,15 +122,21 @@ func runServer(sa *serverArgs, printf, fatalf shared.FormatFn) {
 	// get aspect registry with proper aspect --> api mappings
 	eval := expr.NewCEXLEvaluator()
 	adapterMgr := adapterManager.NewManager(adapter.Inventory(), aspect.Inventory(), eval, gp, adapterGP)
+	var store config.KVStore
+	store, err = config.NewStore(sa.configStoreURL)
+	if err != nil {
+		fatalf("%s", err.Error())
+	}
 	configManager := config.NewManager(eval, adapterMgr.AspectValidatorFinder, adapterMgr.BuilderValidatorFinder,
 		adapterMgr.SupportedKinds,
-		sa.globalConfigFile, sa.serviceConfigFile, time.Second*time.Duration(sa.configFetchIntervalSec))
+		store, time.Second*time.Duration(sa.configFetchIntervalSec))
 
 	var serverCert *tls.Certificate
 	var clientCerts *x509.CertPool
 
 	if sa.serverCertFile != "" && sa.serverKeyFile != "" {
-		sc, err := tls.LoadX509KeyPair(sa.serverCertFile, sa.serverKeyFile)
+		var sc tls.Certificate
```

Comment:
These certs just are there, provided by the operator. There is no cert management beyond the ability to specify them on the command-line. I expect this stuff to be removed once we have a true auth story.